src/Common/src/Microsoft/Internal/Assumes.InternalErrorException.cs

```diff
@@ -15,7 +15,7 @@ internal partial class Assumes
         private sealed class InternalErrorException : Exception
         {
             public InternalErrorException(string message)
-                : base(string.Format(CultureInfo.CurrentCulture, CommonStrings.Diagnostic_InternalExceptionMessage, message))
```

Comment:
You shouldn't change this files as this will break other projects that use CommonResources.